nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -103,6 +100,14 @@ def test_plot_functions():
         with tempfile.TemporaryFile(suffix='.png') as fp:
             plot_func(img, output_file=fp.name)
 
+    # test for bad input arguments (cf. #510)
+    ax = pl.subplot(111, rasterized=True)
+    out = plot_stat_map(
+        img, symmetric_cbar=True,
+        output_file=tempfile.TemporaryFile(suffix='.png'),
```

Comment:
Can you use a `with tempfile.TemporaryFile` as above? IIRC this will cause a warning with Python 3.